pkg/services/gitops/manifests/app-crd.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
It seems odd that this is now in the `pkg` directory. Shouldn't this be in `config/crd`?

Reply:
indeed, I'll update to get your changes.